pkg/cli/admin/release/info.go

```diff
@@ -744,7 +744,10 @@ func (i *ReleaseInfo) Platform() string {
 	if len(arch) == 0 {
 		arch = "unknown"
 	}
-	return fmt.Sprintf("%s/%s", os, arch)
+	if len(i.ManifestListDigest) == 0 {
+		return fmt.Sprintf("%s/%s", os, arch)
+	}
+	return fmt.Sprintf("multi (%s/%s)", os, arch)
```

Comment:
nit: adding a third `if` seems like a good opportunity to add a unittest for this method but I do not insist

Reply:
Unit tests added in aa87d2528 -> ae460964e, and they inspired me to add a `nil`-`Config` guard in here, so that's helpful.

Reply:
/hold cancel